### PR TITLE
Fix - issue #953

### PR DESCRIPTION
**Pull request by BhargavBhandari90**
Medium image not showing with zero height in activity

### CODE REVIEW

File: app/main/RTMedia.php [hunk 1]
```diff
@@ -301,10 +301,28 @@ function custom_style_for_image_size() {
 	}
 
 	function custom_style_for_activity_image_size() {
+
+		// get width from rtMedia settings.
```
Comment (kiranpotphode) [inline app/main/RTMedia.php]: First letter capital for inline comments.

File: app/main/RTMedia.php [hunk 2]
```diff
@@ -301,10 +301,28 @@ function custom_style_for_image_size() {
...
+		$width  = ( isset( $this->options['defaultSizes_photo_medium_width'] ) ) ? $this->options['defaultSizes_photo_medium_width'] : '0';
+		// get height from rtMedia settings.
```
Comment (kiranpotphode) [inline app/main/RTMedia.php]: Start with first letter capital for inline comments.

**Comment by kiranpotphode** [comment]
@BhargavBhandari90 requested some minor changes.

**Comment by BhargavBhandari90** [comment]
@pradeep910 Please do the final review.

**Pull request merged**